diff --git a/usr/lib/hwsupport/power-button-handler.py b/usr/lib/hwsupport/power-button-handler.py
index 1a46251..cbf7857 100755
--- a/usr/lib/hwsupport/power-button-handler.py
+++ b/usr/lib/hwsupport/power-button-handler.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/bin/env python3 -u

diff --git a/usr/share/jupiter-fan-control/fancontrol.py b/usr/share/jupiter-fan-control/fancontrol.py
index 1a46251..cbf7857 100755
--- a/usr/share/jupiter-fan-control/fancontrol.py
+++ b/usr/share/jupiter-fan-control/fancontrol.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python -u
+#!/bin/env python3 -u
 """jupiter-fan-controller"""
 import signal
 import os

diff --git a/usr/share/jupiter-fan-control/PID.py b/usr/share/jupiter-fan-control/PID.py
index 1a46251..cbf7857 100755
--- a/usr/share/jupiter-fan-control/PID.py
+++ b/usr/share/jupiter-fan-control/PID.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/bin/env python3
 #
 # This file is part of IvPID.
 # Copyright (C) 2015 Ivmech Mechatronics Ltd. <bilgi@ivmech.com>
